### PR TITLE
Ft allow fellows comment on logged activity#159101182

### PR DESCRIPTION
**Pull request by andela-fomokaro**
## Description
As a logged in user,
- WHEN I am on the My activities page.
- AND there is a logged activity that requires additional details.
- THEN I should be able to click on the activity card to add more information.

## How to test
Once logged in as a user (Fellow)
- Navigate to My Activities page.
- Click on any card that has the status of (in review or pending).
- Ensure the cards with the statuses above are clickable.
- Ensure a modal pops up containing the activity information
- Fill in the required information on the activity card.
- Click on the update button. Ensure messages are correctly seen and the modal closes.

## Screenshots

screen shot 2018-07-27 at 4 28 25 pm

### CODE REVIEW

File: src/containers/MyActivities.jsx [hunk 1]
```diff
@@ -87,6 +92,47 @@ class MyActivities extends Component {
...
+    }
+    return null;
+  }
+
+  updateSelectedActivity = (newValues) => {
+    const {
+      date,
+      category,
+      description,
+      noOf,
```
Comment (mkiterian) [inline src/containers/MyActivities.jsx]: `noOf` is `numberOf` in the LogActvityForm state. Does this still work?

File: src/components/activities/ActivityCard.jsx [hunk 2]
```diff
@@ -99,9 +99,10 @@ class ActivityCard extends Component {
 
   constructor(props) {
     super(props);
+    const getStatuses = Object.keys(STATUSES).map(key => (STATUSES[key]));
```
Comment (kimani-ndegwa) [inline src/components/activities/ActivityCard.jsx]: Why not just call it `statuses` and avoid defining a value when setting state?

Comment (andela-fomokaro) [reply src/components/activities/ActivityCard.jsx]: @kimani-ndegwa done

**Comment by andela-fomokaro** [comment]
@kimani-ndegwa Overall coverage increased (+0.003%) to 87.121% 🙂

**Comment by andela-fomokaro** [comment]
@kimani-ndegwa Please don't merge this PR yet. I have a test line to push

**Comment by andela-fomokaro** [comment]
@kimani-ndegwa PR ready.

**Comment by kimani-ndegwa** [comment]
Please rebase @andela-fomokaro

**Comment by andela-fomokaro** [comment]
@kimani-ndegwa all done

**Pull request closed without merging**